pkg/chunked/storage_unsupported.go

```diff
@@ -15,3 +15,9 @@ import (
 func GetDiffer(ctx context.Context, store storage.Store, blobSize int64, annotations map[string]string, iss ImageSourceSeekable) (graphdriver.Differ, error) {
 	return nil, errors.New("format not supported on this architecture")
 }
+
+// GetTOCDigest returns the digest of the TOC as recorded in the annotations.
+// This is an experimental feature and may be changed/removed in the future.
+func GetTOCDigest(annotations map[string]string) (*digest.Digest, error) {
+	return nil, errors.New("format not supported on this architecture")
```

Comment:
Looking around in this package, this should be complaining about the OS.

Reply:
changed the error message to the more generic "format not support on this system"